css/index.css

```diff
@@ -100,6 +129,136 @@ p {
     flex-wrap: wrap;
     justify-content: space-evenly;
 }
+.box1 {
+    width: 12.5%;
+    height: 100px;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: teal;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+}
+.box2 {
+    width: 12.5%;
+    height: 100px;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    background-color: gold;
+}
```

Comment:
you gave yourself extra work repeating all the styling for each box (consider one box class, and then `box1`, `box2`, `box3` etc. for colors)